src/components/editableText/EditableText.js

```diff
@@ -29,24 +29,47 @@ export class EditableText extends Component {
     });
   }
 
+  renderEditableControl() {
+    const {
+      onUpdateFieldHandle,
+      width,
+      multirow
+    } = this.props;
+    const { onChange, updateText } = this;
+    const { editValue } = this.state;
+    const editableControl = !multirow ? (
+      <input
+        autoFocus
+        type="text"
+        onChange={onChange}
+        onBlur={() => updateText(onUpdateFieldHandle)}
```

Comment:
do not pass `onUpdateFieldHandle` to `updateText`, you can get it directly from `updateText` _this.props.onUpdateFieldHandle_. The same for `onBlur` on textarea